client/components/layout/Container/index.js

```diff
@@ -0,0 +1,14 @@
+import React, { PropTypes } from 'react'
+
+import Base from '../internal/Base'
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string
+}
```

Comment:
This list should match that of all of the other components.  Another argument for only validating them on `Base` perhaps?